.travis-setup.sh

```diff
@@ -17,42 +17,49 @@
 #  along with this program; if not, write to the Free Software
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-glib_version=2.49.4
+set -e -x
+
+glib_version=2.46.2
```

Comment:
The intention was that we do not introduce the use of newer symbols/defines without the mandatory pkg-config check in the configure.ac.

It's too easy to use a new function and not bump the dependency in the configure.ac and travis can help us catch that.